queries/orderbook/barn_batch_rewards.sql

```diff
@@ -273,16 +286,37 @@ primary_rewards as (
     GROUP BY
         solver
 ),
+integrator_fees_per_solver AS (
```

Comment:
I guess renaming `integrator` to `partner` everywhere would make it a bit cleaner. Not a strong opinion though.

Reply:
Replaced every instance of the word `integrator` with `partner`.